PrivateHospitals/backend/PrivateHospitals/PrivateHospitals.Infrastructure/Repositories/Statistics/StatisticsRepository.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Dapper;
+using PrivateHospitals.Core.Models.Statistics;
+using PrivateHospitals.Infrastructure.Interfaces.Statistics;
+using PrivateHospitals.Infrastructure.Loader;
+using System.Data;
+
+
+namespace PrivateHospitals.Infrastructure.Repositories.Statistics
+{
+    public class StatisticsRepository : IStatisticsRepository
+    {
+        private readonly IDbConnection _dbConnection;
+        private readonly SqlQueryLoader _queryLoader;
+
+        public StatisticsRepository(IDbConnection dbConnection)
+        {
+            _dbConnection = dbConnection;
+            _queryLoader = new SqlQueryLoader();
```

Comment:
DI violation. U will need to remove it at all when u use SqlResources which I mentioned